Thumbprint/Targets/Thumbprint/Components/CalendarPicker/DayOfWeekHeaderView.swift

```diff
@@ -18,7 +18,7 @@ class DayOfWeekHeaderView: UIView {
             return label
         })
 
-        stackView = UIStackView(arrangedSubviews: labels)
+        self.stackView = UIStackView(arrangedSubviews: labels)
```

Comment:
This change made me curious - what's up? I don't see a local declaration or anything - why disambiguate with self here?